templates/index.html

```diff
@@ -57,8 +57,14 @@ <h3 class="fs-title">
 </div>
 
 {% if CASS_HOME_EXTRAS %}
-    {% set home_extras = pages[7] %}  <!-- Set to 6 to grab homepage extras --> 
+    {% set home_extras = pages[7] %}  <!-- Set to 7 to grab homepage extras --> 
     {{ home_extras.content }}
 {% endif %}
 
+{% if OSL_HOME_EXTRAS %}
+    {% set home_extras = pages[4] %}  <!-- Set to 4 to grab homepage extras --> 
```

Comment:
Content is now defined by a ``frontpage`` tag. I could add a loop.index restriction to limit the number of pages that pelican will draw content from?

If ``:date:`` metadata is defined at the top of the file, only the most recent content will be displayed. Otherwise, I think it grabs the first page from an alphabetical list.